vertical-pod-autoscaler/pkg/utils/vpablessing/vpa_blessing.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vpablessing
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+const (
+	// VpaBlessingLabel is a label used by the vpa blessing annotation.
+	VpaBlessingLabel = "containersObservedByAdmissionController"
```

Comment:
AdmissionController is a general name, so this does not hint that VPA creates this annotation.
Since the current annotation that we already add is named vpaUpdates, maybe vpaObservedContainers?

Reply:
"vpaObservedContainers" it is.